schematic/schemas/data_model_relationships.py

```diff
@@ -188,6 +191,16 @@ def define_data_model_relationships(self) -> dict:
                     "standard": get_label_from_display_name,
                 },
             },
+            "columnType": {
+                "jsonld_key": "sms:columnType",
+                "csv_header": "ColumnType",
+                "node_label": "columnType",
+                "type": str,
+                "required_header": False,
+                "edge_rel": False,
+                "node_attr_dict": {"default": None},
+                "allowed_values": ["string", "integer", "number", "boolean"],
```

Comment:
Susheel was mentioning something like using xkd? to link the values?  I wonder if that's of value for this ticket if we're largely unsure what the json-ld is going to look like in October?

Reply:
Yep! I'm happy to do it if we can find the right ontology. I'm good at getting it out there without it for now and adding it in leter if its important enough.